packages/calcite-components/src/components/tile-group/tile-group.tsx

```diff
@@ -0,0 +1,94 @@
+import { Component, Element, h, Prop, VNode, Watch } from "@stencil/core";
+import {
+  connectInteractive,
+  disconnectInteractive,
+  InteractiveComponent,
+  InteractiveContainer,
+  updateHostInteraction,
+} from "../../utils/interactive";
+import { TileGroupLayout } from "./interfaces";
+import { Scale } from "../interfaces";
+import { CSS } from "./resources";
+
+/**
+ * @slot - A slot for adding `calcite-tile` elements.
+ */
+@Component({
+  tag: "calcite-tile-group",
+  styleUrl: "tile-group.scss",
+  shadow: true,
+})
+export class TileGroup implements InteractiveComponent {
+  //--------------------------------------------------------------------------
+  //
+  //  Properties
+  //
+  //--------------------------------------------------------------------------
+
+  /** When `true`, interaction is prevented and the component is displayed with lower opacity. */
+  @Prop({ reflect: true }) disabled = false;
+
+  /**
+   * Defines the layout of the component.
+   *
+   * Use `"horizontal"` for rows, and `"vertical"` for a single column.
+   */
+  @Prop({ reflect: true }) layout: TileGroupLayout = "horizontal";
+
+  /**
+   * Specifies the size of the component.
+   */
+  @Prop({ reflect: true }) scale: Scale = "m";
+
+  @Watch("scale")
+  scaleWatcher(): void {
+    this.updateTiles();
+  }
+
+  //--------------------------------------------------------------------------
+  //
+  //  Private Properties
+  //
+  //--------------------------------------------------------------------------
+
+  @Element() el: HTMLCalciteTileGroupElement;
+
+  //--------------------------------------------------------------------------
+  //
+  //  Private Methods
+  //
+  //--------------------------------------------------------------------------
+
+  private updateTiles = (): void => {
+    this.el.querySelectorAll("calcite-tile").forEach((item) => (item.scale = this.scale));
```

Comment:
Maybe for future but can we get the slotted assigned elements instead of querying the dom?